content/ny-funksjonalitet/roadmap/altinn3/sluttbruker.md

```diff
@@ -12,7 +12,9 @@ For informasjon om **hvordan man bruker** funksjonalitetene som er beskrevet her
 {{% /panel %}}
 ### Skjemafunksjonalitet
 
-Støtte for innsendingstjenester. Et utvalg standardkomponenter å bygge med (eks. tekstfelt, radioknapp, sjekkliste, datovelger, vedlegg). Standarddesign der alle komponenter kommer under hverandre. Støtte for at skjemaet består av flere sider, men avgrenset til at sider kommer i bestemt rekkefølge.
+Støtte for innsendingstjenester. Et utvalg standardkomponenter å bygge med (eks. tekstfelt, radioknapp, sjekkliste, datovelger, vedlegg), inkludert sidestilling av elementer. Støtte for at skjemaet består av flere sider, men avgrenset til at sider kommer i bestemt rekkefølge. Alle skjema og elementer har standardisert look and feel.
```

Comment:
"avgrenset til at sider kommer i bestemt rekkefølge" Vi har vel i praksis nå støtte for dynamikk for rekkefølge på sider?

Reply:
Legg til en parates `(grid)`etter "sidestilling av elementer", slik at utviklere med en gang skjønner hva det er snakk om.